app/components/marketplace-plugin-list/constants.js

```diff
@@ -35,5 +35,40 @@ $ appknox upload &lt;path to apk/ipa file&gt; | xargs appknox cicheck --risk-thr
 </pre>
       </li>
     </ul>
-  `
+  `,
+appCenterInstructions:`
+  <ul class="bullet-list margin-r-1 margin-l-1h">
+    <li class="margin-b-1">
+      Generate a personal access token from <a href="/settings/developersettings" target="_blank">Developer Settings</a>
+    </li>
+    <li class="margin-b-1">
+      Create a file <code class="black-text">appcenter-post-build.sh</code> in your source code repository and add the content given below:
```

Comment:
Should we specify in what directory the file needs to be created? E.g. for Android its: `/app/appcenter-post-build.sh` and for IOS it's `/appcenter-post-build.sh`.

Reply:
Yes. that will give more clarity. Have done the change.